src/components/encrypt-video.js

```diff
@@ -9,9 +9,14 @@ module.exports = React.createClass({
   },
   getInitialState() {
     return {
-      seventyFivePercentDone: false
+      nextPercentDone: 0.25
     };
   },
+  videoGaEvent(props) {
+    props.category = props.category || "Video";
+    props.label = props.label || this.context.intl.formatMessage({id: this.props.video.title});
```

Comment:
What happens if the video title isn't localized? Most likely scenario seems to be title is localized, video isn't, or neither are localized and user in another region is seeing English title, which might affect the en-US stats. Not necessarily a blocker here, just want to point it out.